jekyll/_cci2/adv-config.md

```diff
@@ -87,8 +206,96 @@ For more information on browser testing, see the [Browser Testing]({{site.baseur
 
 Use a service container to run database testing:
 
+{:.tab.executors_four.Cloud}
+``` yaml
+version: 2.1
+
+orbs: 
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+
+    # Primary container image where all commands run
+    docker:
+      - image: cimg/python:3.6.2-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+        environment:
+          TEST_DATABASE_URL: postgresql://root@localhost/circle_test
+
+    # Service container image
+      - image: cimg/postgres:9.6.5
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+
+    steps:
+      - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install postgresql-client-9.6
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`